[RA2 Ch2]: Defining the Kubernetes NFPG Network Attachment API as mandatory (#3209)
cntt-n/CNTT: doc/ref_arch/kubernetes/chapters/chapter03.rst

`SR-IOV Network Device Plugin <https://github.com/k8snetworkplumbingwg/sriov-network-device-plugin>`__, that discovers
and advertises SR-IOV Virtual Functions (VFs) available on a Kubernetes node, and is used to map VFs to scheduled pods.
To use it, the SR-IOV CNI is required, as well as a CNI multiplexer plugin (such as
`Multus CNI <https://github.com/k8snetworkplumbingwg/multus-cni>`__), to provision additional secondary network

> interfaces for VFs (beyond the primary network interface). The SR-IOV CNI during pod creation allocates a SR-IOV VF to a

Reviewer: ```suggestion
interfaces for VFs (beyond the primary network interface). The SR-IOV CNI, during pod creation, allocates an SR-IOV VF to a
```

differences between networking solutions that consist of these options. It is important to note that different
networking solutions require different descriptors from the Kubernetes workloads (specifically, the deployment
artefacts such as YAML files, etc.), therefore the networking solution should be agreed between the CNF vendors and the
CNF operators. To allow easy integration of CNF-s and the platforms it is recommended to use either the plain CNI API
resources or the API resources defined in the v1.2 of the `Kubernetes Network Custom Resource Definition De-facto
Standard <https://github.com/k8snetworkplumbingwg/multi-net-spec/tree/master/v1.2>`__.

-  The **Default CNI Plugin** through the use of deployment specific configuration (e.g. `Tungsten Fabric
   <https://tungstenfabric.github.io/website/Tungsten-Fabric-Architecture.html#vrouter-deployment-options>`__)
-  A **multiplexer/meta-plugin** that integrates with the Kubernetes control plane

>    via CNI (Container Network Interface) and allows for use of multiple CNI plugins

Reviewer: ```suggestion
   via CNI (Container Network Interface) and allows the use of multiple CNI plugins
```

>    in order to provide this specific connectivity that the default Network Plugin may

Reviewer: what specific connectivity?
what is the "the default Network Plugin"?

Author: The default Network Plugin is the default CNI of the cluster. 
The specific connectivity is to have multiple network connections to a Pod.  Should I try to rephrase this?

Reviewer: @CsatariGergely I think you fixed it with the new wording. Thanks

>    not be able to provide (e.g. `Multus <https://github.com/intel/multus-cni>`__)

Reviewer: Multus is an example of the multiplexer/meta-plugin and not the default network CNI plugin
```suggestion
   not be able to provide.
```

   the Pod. Note that the different network characteristics of the interfaces might
   require different networking technologies, which would potentially require
   different CNI plugins. Also note that this is only required for the High Performance

>    profile. Example CNI implementations which meet these requirements is

Reviewer: since only one example plugin is being listed
```suggestion
   profile. An example CNI implementation that meets these requirements is the
```
